src/main/proto/ga4gh/reads.proto

```diff
@@ -31,37 +31,40 @@ message ReadGroup {
 
   // The ID of the dataset this read group belongs to.
   string dataset_id = 2;
+  
+  // The ID of the read group set this read group belongs to.
+  string read_group_set_id = 3;
 
   // The read group name.
-  string name = 3;
+  string name = 4;
```

Comment:
You should almost never reuse existing tag numbers: https://developers.google.com/protocol-buffers/docs/proto3#assigning-tags

Technically we could decide to be more nitpicky about enforcing this rule; e.g. it's OK to change tag numbers which haven't yet made it into a formal release. In my experience, it's easier to not try to think of all the possible scenarios where a tag might already be in use by a client somewhere, and just disallow reusing tag numbers on master.